Client/Frontend/Home/HomePanelViewController.swift

```diff
@@ -136,7 +136,7 @@ class HomePanelViewController: UIViewController, UITextFieldDelegate, HomePanelD
         buttonContainerView.addGestureRecognizer(dismissKeyboardGestureRecognizer)
 
         // Invalidate our activity stream data sources whenever we open up the home panels
```

Comment:
This comment is no longer true. If anything, we check to see if we need to refresh Top Sites.